### PR TITLE
62 hamburger menu icon styling adjustment

### PR DESCRIPTION
**Pull request by GeorgeSwift16**

### CODE REVIEW

**Review by DanForder — CHANGES_REQUESTED** [review]
- Fix tests, merge from main
- Do we not already have those SVGs? we demo-ed them today

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import AppLogo from "../../assets/images/AppLogo.svg";
 import BackButton from "../../assets/images/BackButton.svg";
 import "./Header.scss";
+import MenuIcon from "../MenuIcon/MenuIcon";
+
+const Header = ({ toggleMenu }) => {
+  // const menuIconString = menu.toString();
+  // <img src={menuIconString} alt="An icon of a menu" />
```
Comment (DanForder) [inline src/components/Header/Header.jsx:8]: remove commented code

File: src/components/Menu/Menu.scss [hunk 2]
```diff
@@ -1,35 +1,46 @@
 @use "../../styles/variables" as *;
 @use "../../styles/mixins" as *;
 
+.layout__menu {
```
Comment (DanForder) [inline src/components/Menu/Menu.scss:4]: this style should go in `layout.scss`

File: src/components/Menu/Menu.scss [hunk 3]
```diff
@@ -1,35 +1,46 @@
...
+  height: 83%;
+}
+
 .menu {
   position: absolute;
   z-index: 100000;
   width: 90%;
+  & > div:nth-child(even) {
```
Comment (DanForder) [inline src/components/Menu/Menu.scss:12]: does this need to be a div? use BEM if poss

File: src/components/MenuIcon/MenuIcon.scss [hunk 4]
```diff
@@ -0,0 +1,8 @@
+@use "../../styles/variables" as *;
+
+.hamburger-menu {
```
Comment (DanForder) [inline src/components/MenuIcon/MenuIcon.scss:3]: Block class should match the component name: `menu-icon`

**Review by DanForder — CHANGES_REQUESTED** [review]

**Review by DanForder — APPROVED** [review]

**Review by Cris2508 — CHANGES_REQUESTED** [review]
Hi George. could you please change the style of the hamburger menu? It needs to be rectangle and at the moment is square.

**Review by Cris2508 — APPROVED** [review]

**Pull request merged**